vagrant-ansible/strichliste-backend.yml

```diff
@@ -0,0 +1,77 @@
+---
+- hosts: all
+  gather_facts: False
+  become: True
+  become_user: root
+  tasks:
+      - name: install dependencies
+        action: apt pkg={{ item }} state=present
+        with_items:
```

Comment:
Ansible >= 2.7 has a better readable syntax. You can pass a list for `pkg=`, no need to use `with_items` like before.